### DISCUSSION TITLE
ui embed.go pattern all:dist: no matching files found

### DISCUSSION THREAD

**Discussion by maxneo4** [pocketbase/pocketbase#1446] [Q&A] [answered]
I'm trying to use pocket base as framework, although I had the error:
`\github.com\pocketbase\pocketbase@v0.10.4\ui\embed.go:10:12: pattern all:dist: no matching files found `

I only need to run basic main example with framework, after I will test with flags to change the port, so I will get to put pocketbase running in Azure webApp.

First I got install pocket base using the help of "Goland 2021.2.4", but when I run the main file (using as directory, as main file, etc) I always got the same error. I reviewed downloaded folders and I could see dist folder was there.

Second I tried configuring a new empty go project. This time the error happens when I launched the command: 
 `go get github.com/pocketbase/pocketbase`

I share the steps I did with new empyt golang project to install and use the basic main example;

* main.go code:
```go
package main

import (
	"github.com/pocketbase/pocketbase"
	"log"
)

func main() {
	app := pocketbase.New()

	if err := app.Start(); err != nil {
		log.Fatal(err)
	}
}
```
* command line steps

```cmd
PS H:\gitP\pocketbase-azure> go mod init pocketbase.io.azure
go: creating new go.mod: module pocketbase.io.azure
go: to add module requirements and sums:
        go mod tidy
PS H:\gitP\pocketbase-azure> go build
main.go:4:2: no required module provides package github.com/pocketbase/pocketbase; to add it:
        go get github.com/pocketbase/pocketbase
PS H:\gitP\pocketbase-azure> go get github.com/pocketbase/pocketbase
C:\Users\maxne\go\pkg\mod\github.com\pocketbase\pocketbase@v0.10.4\ui\embed.go:10:12: pattern all:dist: no matching files found

```

**Environment Information**

**Os:** Windows 10 pro
**Go version:**  go1.17.2 windows/amd64

**Comment by ganigeorgiev** [comment]
PocketBase requires Go 1.18+.

For instructions how to upgrade your Go installation, please check - https://go.dev/doc/install.

**Comment by maxneo4** [comment, marked as answer]
It works, I upgraded to go1.19.4 windows/amd64 and main base example runs ok.
thanks so much

**Comment by maxneo4** [comment]
I upgraded go version to latest and this work fine. Additional I have an issue to serve static files but in other discussion I saw the answer. Thanks so much.
